lib/pg_eventstore/commands/append.rb

```diff
@@ -9,17 +9,18 @@ class Append < AbstractCommand
       # @param options [Hash]
       # @option options [Integer] :expected_revision provide your own revision number
       # @option options [Symbol] :expected_revision provide one of next values: :any, :no_stream or :stream_exists
+      # @param event_modifier [#call]
       # @return [Array<PgEventstore::Event>] persisted events
       # @raise [PgEventstore::WrongExpectedRevisionError]
-      def call(stream, *events, options: {})
+      def call(stream, *events, options: {}, event_modifier: EventModifiers::PrepareRegularEvent)
```

Comment:
Not sure if it is best to name it as this class is not a modeling event it is just built in a proper way depends if it is a link or regular. 
Maybe we can think about `Event::RegularEventData` `Event::LinkEventData` and the attr: `eventdata: Event::RegularEventData`

Reply:
Well, it is actually returns an event - not the event data.